# Service that can be used to control the robot joint positions using moveit.

trajectory_msgs/JointTrajectoryPoint point
---
bool success
string message